llvm/tools/llvm-exegesis/llvm-exegesis.cpp

```diff
@@ -291,6 +297,22 @@ T ExitOnFileError(const Twine &FileName, Expected<T> &&E) {
   return std::move(*E);
 }
 
+static const char *getIgnoredOpcodeReasonOrNull(const LLVMState &State,
```

Comment:
I can't find a function with similar logic. Can you please tell me its name?

Reply:
Here: https://github.com/llvm/llvm-project/blob/79643565a890d29560782fea23877ae61ea5b987/llvm/tools/llvm-exegesis/lib/SerialSnippetGenerator.cpp#L58

Reply:
Removed this check from tools/llvm-exegesis/lib/SerialSnippetGenerator.cpp, since the program always first goes to generateSnippets in the file tools/llvm-exegesis/llvm-exegesis.cpp, where the same check already done and returns an error if it fails